plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/CubridTable.java

```diff
@@ -115,4 +127,28 @@ public Object[] getPossibleValues(CubridTable object)
             return object.getDataSource().getSchemas().toArray();
         }
     }
+    
+    class PartitionCache extends JDBCObjectCache<CubridTable, CubridPartition>{
+
+        @Override
+        protected JDBCStatement prepareObjectsStatement(JDBCSession session, CubridTable table)
+            throws SQLException {
+            String sql = "select * from db_partition where class_name = ? and owner_name = ?";
+            final JDBCPreparedStatement dbStat = session.prepareStatement(sql);
+            dbStat.setString(1, table.getName());
+            dbStat.setString(2, table.getSchema().getName());
+            return dbStat;
+        }
+    
+        @Override
+        protected CubridPartition fetchObject(
+            JDBCSession session, CubridTable table, JDBCResultSet dbResult)
+            throws SQLException, DBException {
+            String partition_class_name = JDBCUtils.safeGetString(dbResult, "partition_class_name");
+            String type = JDBCUtils.safeGetString(dbResult, "partition_type");
+            
+            return new CubridPartition(table, partition_class_name, type, dbResult);
+        }
+            
+        }

```

Comment:
Please fix formatting
```suggestion
    class PartitionCache extends JDBCObjectCache<CubridTable, CubridPartition>{

        @Override
        protected JDBCStatement prepareObjectsStatement(@NotNull JDBCSession session, @NotNull CubridTable table) throws SQLException {
            String sql = "select * from db_partition where class_name = ? and owner_name = ?";
            final JDBCPreparedStatement dbStat = session.prepareStatement(sql);
            dbStat.setString(1, table.getName());
            dbStat.setString(2, table.getSchema().getName());
            return dbStat;
        }
    
        @Override
        protected CubridPartition fetchObject(
            @NotNull JDBCSession session,
            @NotNull CubridTable table,
            @NotNull JDBCResultSet dbResult
        ) throws SQLException, DBException {
            String partition_class_name = JDBCUtils.safeGetString(dbResult, "partition_class_name");
            String type = JDBCUtils.safeGetString(dbResult, "partition_type");
            
            return new CubridPartition(table, partition_class_name, type, dbResult);
        }
            
    }

```